Formula/ec2instanceconnect.rb

```diff
@@ -0,0 +1,74 @@
+class Ec2instanceconnect < Formula
+  include Language::Python::Virtualenv
+
+  desc "All-in-one client for EC2 Instance Connect"
+  homepage "https://github.com/aws/aws-ec2-instance-connect-cli"
+  url "https://files.pythonhosted.org/packages/0c/7b/2dbd14b2caf0d834f781ef982bb4b22884245c44af0ebb6ccc5a7019fd3a/ec2instanceconnectcli-1.0.0.tar.gz"
+  sha256 "dc818c78d51a475a97d502db0221962e3c42af66900a63d7da69d3d68af29de8"
+  revision 1
```

Comment:
```suggestion
```

Reply:
For the brand new version, we dont need `revision` line.

Reply:
Removed